wordpress-mobile/gutenberg-mobile
# Add CI Check for package-lock.json diffs

This PR adds a CI check for changes on `package-lock.json` after running `npm install`.
With this check we will ensure that our `package-lock.json` is always updated and changes are committed.

To test:
- Check CI results for `check-correctness` job
- Check [this previous run](https://app.circleci.com/pipelines/github/wordpress-mobile/gutenberg-mobile/8397/workflows/5a4ea911-4f44-40ef-81a7-33bdb3386bf1/jobs/44605) job with this check failing to see output.
- Feel free to do other tests as desired.

PR submission checklist:
